cartography/intel/aws/secretsmanager.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from cartography.util import dict_date_to_epoch
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_secret_list(boto3_session: boto3.session.Session) -> List[Dict]:
+    client = boto3_session.client('secretsmanager')
+    paginator = client.get_paginator('list_secrets')
+    secrets: List[Any] = []
+    for page in paginator.paginate():
+        secrets.extend(page['SecretList'])
+    return secrets
+
+
+@timeit
+def load_secrets(neo4j_session: neo4j.Session, data: List, current_aws_account_id: str, aws_update_tag: int) -> None:
+    ingest_secrets = """
+    UNWIND {Secrets} as secret
+        MERGE (s:SecretsManagerSecret{id: secret.ARN})
+        ON CREATE SET s.firstseen = timestamp()
+        SET s.name = secret.Name, s.description = secret.Description, s.kms_key_id = secret.KmsKeyId,
+            s.rotation_enabled = secret.RotationEnabled, s.rotation_lambda_arn = secret.RotationLambdaARN,
+            s.rotation_rules_automatically_after_days = secret.rotation_rules_automatically_after_days,
+            s.last_rotated_date = secret.LastRotatedDate, s.last_changed_date = secret.LastChangedDate,
+            s.last_accessed_date = secret.LastAccessedDate, s.deleted_date = secret.DeletedDate,
+            s.owning_service = secret.OwningService, s.created_date = secret.CreatedDate,
+            s.primary_region = secret.PrimaryRegion,
+            s.lastupdated = {aws_update_tag}
+        WITH s
+        MATCH (owner:AWSAccount{id: {AWS_ACCOUNT_ID}})
+        MERGE (owner)-[r:RESOURCE]->(s)
+        ON CREATE SET r.firstseen = timestamp()
+        SET r.lastupdated = {aws_update_tag}
+    """
+    for secret in data:
+        secret['rotation_rules_automatically_after_days'] = secret.get(
+            'RotationRules', {},
+        ).get('AutomaticallyAfterDays')
```

Comment:
So I understand the structure is something like this through the boto3 documentation: 
`'RotationRules': {
                'AutomaticallyAfterDays': 123
            },`
Are you using the `.get()` function because any one of the two above keys may not have a value? In that case, as far as I know, even if you do `s.rotation_rules_automatically_after_days = secret.RotationRules.AutomaticallyAfterDays` in the Cypher query in `ingest_secrets`, you should be fine. Just test it out once though :)